code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -168,6 +168,8 @@
 	desc = "It's made of a special fiber that provides minor protection against biohazards. This one is in baby blue."
 	icon_state = "scrubsblue"
 	_color = "scrubsblue"
+	permeability_coefficient = 0.50

```

Comment:
The changes you made to this file aren't necessary because all of these values are inherited directly from the parent so they are identical